DOCS/license-matching-guidelines-and-templates.md

```diff
@@ -99,15 +99,15 @@ Punctuation should be matched, unless otherwise stated in these guidelines or un
 
 Any hyphen, dash, en dash, em dash, or other variation should be considered equivalent.
 
-### B.6.4 Guideline: Quotes
+### B.6.4 Guideline: Quotes <a name="B.6.4"></a>
```

Comment:
Is it needed for something?

Reply:
It is represented in other headings (see B.6.3, for example)  and it is used to point to a specific heading (when rendered in HTML).